net-dialup/freeradius-client/freeradius-client-1.1.7-r1.ebuild

```diff
@@ -23,27 +23,25 @@ DOCS=(
 	README.{radexample,rst}
 )
 
+PATCHES=(
+	"${FILESDIR}/${P}-ar-configure.in.patch"
```

Comment:
Convention is to use ${P} for upstreamed patches, so I'll change to ${PN} on merge.